### PR TITLE
add more lmao

### PR DESCRIPTION
**Pull request by zanuka**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
     }
   </style>
   <script>
-    const emotes = ['😂', '🔥', '👌', '💯', '🍞'];
+    const emotes = ['😂', '🔥', '👌', '💯', '🍞', '🐸', '💩', '😆'];
```
Comment (dwats) [inline index.html]: Need to update line 50 otherwise you'll never see the new emojis.

**Review by dwats — CHANGES_REQUESTED** [review]

**Comment by awolden** [comment]
While I am personally a fan of 💩 Is it possible that the usage of this will alienate a large portion of our user base?

**Comment by seanohollaren** [comment]
I think we have to ask which emojis are truly lit enough to appear on the site.

**Comment by cpeddecord** [comment]
we should write a bot that can determine the litness of a given emoji, especially within relative proximity of other emojis.

**Review by dwats — APPROVED** [review]

**Comment by awolden** [comment]
We could have a ML algorithm that scrapes the twitter feed to look for freshest litness.

**Comment by zanuka** [comment]
so much potential here... good ideas guys 🍶 :: feel free to mod this PR ... yes @awolden  💩 good point about this one, no need to alienate. how about some keyboard commands to add/remove lit emojis based on the litness test / twitter scraper? cmd+shift up == 'increase litness factor' cmd+shift down == 'decrease litness factor'

**Pull request closed without merging**